hallux/targets/github_proposal.py

```diff
@@ -118,13 +117,37 @@ def commit_diff(self) -> bool:
                 body=body,
                 commit=commit,
                 side="RIGHT",
-                path=str(compacted.filename),
-                line=compacted.end_line,
-                start_line=compacted.start_line,
+                path=str(self.existing_proposal.filename),
+                line=self.existing_proposal.end_line,
+                start_line=self.existing_proposal.start_line
+                if self.existing_proposal.start_line > self.existing_proposal.end_line
+                else GithubObject.NotSet,
             )
-        except GithubException:
+        except BaseException as ex:
+            logger.debug(ex)
             success = False
 
+        # compacted = self.compact_proposal(self.existing_proposal)
+        #
+        # for line in compacted.proposed_lines:
+        #     body = body + line + "\n"
+        # body = body + "\n```"
+        #
+        #
+        # success: bool = True
+        # try:
+        #     self.pull_request.create_review_comment(
+        #         body=body,
+        #         commit=commit,
+        #         side="RIGHT",
+        #         path=str(compacted.filename),
+        #         line=compacted.end_line,
+        #         start_line=compacted.start_line if compacted.start_line > compacted.end_line else GithubObject.NotSet,
+        #     )
+        # except BaseException as ex:
+        #     logger.debug(ex)
+        #     success = False
+
         # clean local code
         FilesystemTarget.revert_diff(self)
         return success
```

Comment:
Remove this commented out code.
```suggestion
    def commit_diff(self) -> bool:
        commit = self.repo.get_commit(self.pull_request.head.sha)
        body = self.existing_proposal.description + "\n```suggestion\n"

        for line in self.existing_proposal.proposed_lines:
            body = body + line + "\n"
        body = body + "\n```"

        success: bool = True
        try:
            self.pull_request.create_review_comment(
                body=body,
                commit=commit,
                side="RIGHT",
                path=str(self.existing_proposal.filename),
                line=self.existing_proposal.end_line,
                start_line=self.existing_proposal.start_line
                if self.existing_proposal.start_line > self.existing_proposal.end_line
                else GithubObject.NotSet,
            )
        except BaseException as ex:
            logger.debug(ex)
            success = False

        FilesystemTarget.revert_diff(self)
        return success

```